src/renderer/components/server-modal-form.jsx

```diff
@@ -224,15 +227,16 @@ export default class ServerModalForm extends Component {
                   name="name"
                   maxLength="250"
                   placeholder="Name"
-                  value={this.state.name}
+                  value={this.state.name || ''}
                   onChange={::this.handleChange} />
               </div>
               <div className={`six wide field ${this.highlightError('client')}`}>
-                <label>Client</label>
+                <label>Database Type</label>
```

Comment:
Would be good if it had been included in a different commit and pull request.
But I will not make you rebase the PR just because of it. Just keep in mind next time to break down the pull request.